retrieval/programmatic-access/tag_querying.rst

```diff
@@ -122,17 +121,17 @@ table, to see what they apply to.
    * - coastal_brackish:high_confidence
      - coastal_brackish
```

Comment:
why is this presented as 3 bullet points?

Reply:
this formats it to a table (Peter made this document, I just started modifying it)